python/lsst/pipe/tasks/hips.py

```diff
@@ -860,6 +871,46 @@ class GenerateHipsConfig(pipeBase.PipelineTaskConfig, pipelineConnections=Genera
         dtype=float,
         default=8.0,
     )
+    parallel_highest_order = pexConfig.Field[bool](
+        doc=(
+            "If this is set to True, each of the highest order hips pixels will"
+            " be put into their own quanta, and can be run in parallel. The "
+            "trade off is the highest order must be run alone by setting "
+            "min_order to be the the same as the healpix dimension. This will "
+            "skip writing all sky info, which must be done in another"
+            " invocation of this task."
```

Comment:
Please make this consistent about putting line-break spaces at either the end or the beginning.